.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+/nbproject/
```

Comment:
To ignore netbeans files. You should complete with other common rules for PhpStom, Netbeans, Windows thumbnail cache.  See https://github.com/yiisoft/yii2/blob/master/.gitignore as an example.